crates/client/Cargo.toml

```diff
@@ -52,6 +52,8 @@ tiny_http = "0.8"
 url.workspace = true
 util.workspace = true
 worktree.workspace = true
+async-socks5 = "0.6.0"
```

Comment:
We don't want to use `async-socks5`, as it looks like it depends on Tokio.

Zed does not use Tokio, as we have our own async runtime.

Reply:
Well it really is only using some ReadExt and WriteExt traits but not the runtime of Tokio.

Reply:
Let me see how to remove Tokio here.